yarn-completion.plugin.zsh

```diff
@@ -131,8 +129,13 @@ _yc_yarn_commands_completion() {
       'team[Maintain team memberships]' \
       'unlink[Unlink a previously created symlink for a package.]' \
       'upgrade[Upgrades packages to their latest version based on the specified range.]' \
+      'upgrade --latest[Upgrades packages to their latest.]' \
+      'upgrade-interactive[Upgrades selected packages to their latest version based on the specified range.]' \
+      'upgrade-interactive --latest[Upgrades selected packages to their latest version.]' \
```

Comment:
Same idea here...
```suggestion
```

Reply:
I know that it isn't about the available options, but this one is the option that I use the most